### DISCUSSION TITLE
Erweiterung um Seplos Bluetooth

### DISCUSSION THREAD

**Discussion by villingerh** [shining-man/bsc_fw#63] [General]
Hi,

ich habe bei mir 2 Seplos BMS an eine Victron Anlage angeschlossen. Da ich wegen der CAN Verbindung nun die Zellspannungen nicht mehr mit RS485 auslesen kann will ich die mit Bluetooth auslesen.

Ich habe mir da schon mal was in den Code eingebaut und großzügig im Code geändert. Ich kann die Seplos BMS V2 Zellspannungen schon per BT auslesen. Das ist noch im experimentierstadium, da ich manchmal auch Datensalat bekomme.

Da ich die Änderungen gerne zurückgeben will, würde ich mich da gerne mal abstimmen bzgl. Datenmodell.

Da ich aber die Hardware nicht einsetze sondern eigentlich nur die Zellspannungen will, würde ich vielleicht auf der Basis BSC ein BSC Light oder sowas bauen, der nur die Daten per BT einsammelt und per MQTT und Livedatenanzeige bereitstellt.

Dafür aber mit kürzerem Intervall.

Wäre cool sich mal drüber zu unterhalten gerne per mail oder discord.

Grüße,
Harald

**Comment by shining-man** [comment]
Hi,
es gibt Daten, die werden permanent vorgehalten. Dazu gehören Zellspannungen, etc.. Diese findest du in der BmsData.h. Am besten ist es, wenn du hier möglichs alle Daten lieferst. Alle anderen Daten, die en BMS liefert, schiebe ich alle x Sekunden in den MQTT Sendebuffer.

Für mehr schreibe mir einfach eine E-Mail oder schreibe mich auf Discord an.
